### PR TITLE
Reader: fix line-clamp not working in FF

### PR DESCRIPTION
**Pull request by jancavan**
This fixes: https://github.com/Automattic/wp-calypso/issues/6182

**Before (in FF):**
screenshot 2016-06-21 11 27 17

**After (in FF):**
screenshot 2016-06-21 11 26 58

/cc @bluefuton

### CODE REVIEW

**Comment by bluefuton** [comment]
When resizing to a single column of cards in Chrome and FF, it looks like the second line gets faded a little too:

screen shot 2016-06-21 at 20 25 32

**Comment by jancavan** [comment]
Good catch, it's fixed in: https://github.com/Automattic/wp-calypso/pull/6186/commits/1aac68a564f32ef188c300a36e940e9ea5b53bff

I also just reduced the intro paragraph's text and slightly tweaked it to avoid the orphaned text issue: https://github.com/Automattic/wp-calypso/pull/6186/commits/ee7fc0dd472d4fad975cd5be5a73dcdb44d515fc

File: client/reader/start/style.scss [hunk 1]
```diff
@@ -81,14 +81,26 @@
...
 		&:after {
-			display: none;
+			@include long-content-fade;
+			height: 25px;
+			position: absolute;
+				bottom: 0;
+				top: 40px;
+			width: 65%;
+
+			@include breakpoint( ">680px" ) {
```
Comment (bluefuton) [inline client/reader/start/style.scss]: The linter is complaining about this width:

```WARNING: ERROR in breakpoint( >680px ): You can only use these sizes[  480px 660px 960px 1040px ] using the following syntax [ <480px >480px 480px-660px ]
	on line 48 of assets/stylesheets/shared/mixins/_breakpoints.scss
	from line 98 of client/reader/start/style.scss```

Is it alright to change it to 660px?

Comment (jancavan) [inline client/reader/start/style.scss]: My bad, I mistyped it. Fixed in: https://github.com/Automattic/wp-calypso/pull/6186/commits/b553a59ed7d4aea8fa2ca78748ff920caa3d49ca

**Comment by bluefuton** [comment]
Works in FF now and no longer fades the second line 🎉 

Last little tweak - think we're fading out a bit too early, so often half the third line has a fade.

File: client/reader/start/style.scss [hunk 2]
```diff
@@ -81,14 +81,26 @@
...
+		}
 
 		&:after {
-			display: none;
+			@include long-content-fade;
+			height: 25px;
+			position: absolute;
+				bottom: 0;
+				top: 40px;
+			width: 65%;
```
Comment (bluefuton) [inline client/reader/start/style.scss]: Think we're fading too early - can we try, say, 35% width instead of 65%?

Comment (jancavan) [inline client/reader/start/style.scss]: I was trying to conceal the ellipsis that appears on some cards in Chrome, but I guess it isn't too bad: https://github.com/Automattic/wp-calypso/pull/6186/commits/ac0d5417619cae08b87ca56f4f6b26319dd2f751

**Comment by bluefuton** [comment]
🚢 🎉

**Pull request merged**